README.md

```diff
@@ -318,3 +318,5 @@ The Wikisource logo is included as `public/img/Wikisource-logo.svg`,
 as an optimized form of the logo
 found at [commons.wikimedia.org/wiki/File:Wikisource-logo.svg](https://commons.wikimedia.org/wiki/File:Wikisource-logo.svg)
 and subject to the licence restrictions specified on that page.
+
+The PNG version in `resources/Wikisource-logo.svg.png` is based on the same SVG file.
```

Comment:
Is it a straight export, or are there any modifications? Either way, that should be made clear here.

Reply:
It's a straight export from Commons. I've added a link and some more info to the readme.